test/backend/test_model.py

```diff
@@ -27,6 +27,7 @@ def test_relu_node_inplace(self):
       inputs=[helper.make_tensor_value_info("X", TensorProto.FLOAT, [3, 2])],
       outputs=[helper.make_tensor_value_info("X1", TensorProto.FLOAT, [3, 2])])
     tf_rep = prepare(helper.make_model(graph_def))
+
```

Comment:
These changes seem irrelevant, would you mind cleaning up a bit so the commit history will not be unnecessarily long?